tools/markdown_lint/setup.py

```
@@ -1,6 +1,8 @@
 #!/usr/bin/env python3
 """Setup script for the markdown-lint package."""
 
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)